example-context/example-service/example-query/example-query-api/src/main/java/uk/gov/justice/services/example/cakeshop/query/api/RecipesQueryApi.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.services.example.cakeshop.query.api;
+
+import org.slf4j.Logger;
+import uk.gov.justice.services.core.annotation.Handles;
+import uk.gov.justice.services.core.annotation.ServiceComponent;
+import uk.gov.justice.services.core.dispatcher.Requester;
+import uk.gov.justice.services.messaging.JsonEnvelope;
+
+import javax.inject.Inject;
+
+import static org.slf4j.LoggerFactory.getLogger;
+import static uk.gov.justice.services.core.annotation.Component.QUERY_API;
+
+@ServiceComponent(QUERY_API)
+public class RecipesQueryApi {
+
+    private static final Logger LOGGER = getLogger(RecipesQueryApi.class);
+    private static final String FIELD_RECIPE_ID = "recipeId";
+
+    @Inject
+    Requester requester;
+
+    @Handles("cakeshop.queries.recipes")
```

Comment:
all non singular pillar names will need to be refactored once main singular pillar commit has been merged.